### DISCUSSION TITLE
UI

### DISCUSSION THREAD

**Discussion by Nickleaton** [alexjustesen/speedtest-tracker#1352] [Ideas]
Some observations on the UI. 

1. Arrange the graphs as a 2 x 2, so you don't need to scroll down to see the jitter. 
2. The period, eg, the last 24hrs. Should be one control that controls all of them 
3. Add the jitter to one of the 3 current boxes as the top of the dashboard. 
4. New graphs. 
a) Show the average of the current period.

So if you picked 24 hours, you get to see the average of each hour, for the last 2 weeks as an example. That shows you when things are busy during the day. 

If you picked one week, shows you which days are busy. 

All that needs is that period picker for 2, and another picker for actual and average. 

5. One graph. 

Just display one graph with a legend. Upload download with the axis on the left. Ping and jitter with the axis on the right. 

This would be another approach to number 1 above. 

6. Results. Add a clear bad results button that removes old not completed records. 

7. On the run speed test button, I found it confusing I had to hit the button, then hit the speed test to run it. 

But otherwise once I got it sorted, its running very well. Thanks for it.

**Comment by alexjustesen** [comment]
https://github.com/alexjustesen/speedtest-tracker/milestone/5
